test/disk-storage.js

```diff
@@ -3,14 +3,21 @@
 var assert = require('assert')
 var deepEqual = require('deep-equal')
 
-var fs = require('fs')
 var path = require('path')
 var util = require('./_util')
 var multer = require('../')
 var temp = require('fs-temp')
 var rimraf = require('rimraf')
 var FormData = require('form-data')
 
+function assertFileProperties (file, name) {
+  const expectedSize = util.fileSizeByName(name)
+  assert.strictEqual(file.fieldname, path.parse(name).name)
```

Comment:
We should probably have tests where the fieldname and name of the file doesn't match. Otherwise this wouldn't catch if we in the library somehow mix them together 🤔